MapboxVision/src/main/java/com/mapbox/vision/sync/SessionManager.kt

```diff
@@ -199,37 +199,40 @@ internal interface SessionManager {
             override fun onSessionStop(
                 clips: Array<VideoClip>,
                 videoPath: String,
-                outputPath: String,
-                sessionStartMillis: Long
+                cachedTelemetryPath: String,
+                coreSessionStartMillis: Long
             ) {
                 if (currentCountry != Country.Unknown) {
 
-                    val sessionCountryDir = getCountryTelemetryDir() ?: return
-
-                    val sessionCountryDirWithTimeout = "$sessionCountryDir$sessionStartMillis/"
-
-                    moveFromCacheDirToCountryDir(outputPath, sessionCountryDirWithTimeout)
+                    val currentCountryTelemetryPath = getCountryTelemetryDir() ?: return
```

Comment:
maybe rename `getCountryTelemetryDir()` to `getCurrentCountryTelemetryPath()` as it always returns a path for current country

Reply:
Okay